<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-IVQQQE2" ignored="4" name="Test" tests="3" failures="1" timestamp="2021-04-06T14:21:04 IST" time="42.648" errors="0">
  <testcase name="getWeatherInfoFromUI_TC" time="12.763" classname="tests.WeatherInfoFromUITest"/>
  <testcase name="getWeatherInfoFromAPITest" time="3.343" classname="tests.WeatherInfoFromAPITest"/>
  <testcase name="compareWeatherInfoBetweeenAPIAndUI" time="0.017" classname="tests.CompareWeatherInfoTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
null
at tests.CompareWeatherInfoTest.compareWeatherInfoBetweeenAPIAndUI(CompareWeatherInfoTest.java:59)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- compareWeatherInfoBetweeenAPIAndUI -->
  <testcase name="@BeforeSuite extentReportSetUp" time="0.0" classname="generics.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeSuite extentReportSetUp -->
  <testcase name="@BeforeSuite extentReportSetUp" time="0.0" classname="generics.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeSuite extentReportSetUp -->
  <testcase name="@AfterSuite cleanUp" time="0.0" classname="generics.BaseTest">
    <ignored/>
  </testcase> <!-- @AfterSuite cleanUp -->
  <testcase name="@AfterSuite cleanUp" time="0.0" classname="generics.BaseTest">
    <ignored/>
  </testcase> <!-- @AfterSuite cleanUp -->
</testsuite> <!-- Test -->
